### PR TITLE
add suport for dynamodb transact_get_items

### PR DESCRIPTION
**Pull request by cm-iwata**
This pull request add suport for DynamoDB transact_get_items

### CODE REVIEW

File: moto/dynamodb2/responses.py [hunk 1]
```diff
@@ -676,3 +679,69 @@ def describe_time_to_live(self):
...
+                request_id += 1
+                hex_request_id = format(request_id, 'x')
+                err_list.append('com.amazonaws.dynamodb.v20120810.TransactGetItem@%s' % hex_request_id)
+            msg += ', '.join(err_list)
+            msg += "'] at 'transactItems' failed to satisfy constraint: " \
+                   "Member must have length less than or equal to %s" % TRANSACTION_MAX_ITEMS
+
+            return self.error('ValidationException', msg)
+
+        dedup_list = [i for n, i in enumerate(transact_items) if i not in transact_items[n + 1:]]
```
Comment (terrycain) [inline moto/dynamodb2/responses.py]: Add some comments in here with what your doing.

**Comment by terrycain** [comment]
Fix conflicts on the test file and then were good.

**Comment by Dunedan** [comment]
Any update on this? This and support for `transact_write_items` would be very welcome.

**Comment by spulec** [comment]
Added with https://github.com/spulec/moto/pull/2809

**Pull request closed without merging**